cbindgen_no_psl.toml

```diff
@@ -0,0 +1,12 @@
+language = "C"
+include_guard = "DDG_URL_PREDICTOR_H"
+pragma_once = true
+header = """
+// DuckDuckGo URL Predictor - C ABI
+// This header is auto-generated by cbindgen. Do not edit by hand.
+"""
+include_version = true
+braces = "NextLine"
+
+[export]
+exclude = [ "ddg_up_get_psl_data" ]
```

Comment:
This new file is here because I wasn’t able to make cbindgen.toml conditionally exclude `ddg_up_get_psl_data`. I’m open to updating it to a nicer solution if there’s one available.

For now, build scripts should use `cbindgen.toml` or `cbindgen_no_psl.toml` depending on the real-psl feature state.